addon/components/x-toggle/component.js

```diff
@@ -18,17 +26,16 @@ export default Ember.Component.extend({
     'isToggled:x-toggle-container-checked',
     'disabled:x-toggle-container-disabled'
   ],
+  // default value states
+  on: true,
+  off: false,
```

Comment:
@ksnyde The only change I think you need to make are the defaults, since those come from https://github.com/knownasilya/ember-cli-toggle/blob/master/app/components/x-toggle/component.js. Otherwise the tests are broken any way.